Game/Resources/elevatorScene/Scripts/elevator_moveCart.as

```diff
@@ -67,15 +76,39 @@ class elevator_moveCart {
                 tempPos.y += speed * deltaTime;
                 self.SetWorldPosition(tempPos);
                 break;
-        }
+                
+            case 3:
+                if (stopTime < endTime){
+                    stopTime += deltaTime;
+                    tempPos = self.GetWorldPosition();
+                    float s = a * stopTime * stopTime * stopTime / 3 + b * stopTime * stopTime / 2 + c * stopTime;
+                    tempPos.x = startStopPos - s;
+                    self.SetWorldPosition(tempPos);
+                }
+                break;
+            case 4:
+                print("4");
```

Comment:
I guess this is a waiting phase, but we probably shouldn't print alot of stuff when in this state.

Reply:
Ahhh forgot to remove this debugging code!